CHANGES.md

```diff
@@ -1,5 +1,8 @@
-### 0.4.1 (2019-09-05)
+### 0.5.0 (2019-10-17)
+* Adds option to omit the Ion version marker from text output. (#109)
+* Adds support for timestamps with precision beyond microseconds. (#100)
```

Comment:
"beyond microseconds" -- perhaps we can be more precise here...nanoseconds?  infinite precision?